inheritance/src/main/java/inheritance/Theater.java

```diff
@@ -0,0 +1,48 @@
+package inheritance;
+
+import java.util.ArrayList;
+
+public class Theater implements Business {
```

Comment:
This theater class should have a separate set of test for Theater. 🎥 
This class should have a separate set of tests for each method created. Need test methods for  constructor, addReview, toString, viewReviews, addMovie, removeMovie.